src/logion/controllers/locrequest.controller.ts

```diff
@@ -166,11 +166,13 @@ export class LocRequestController extends ApiController {
                 hash: file.hash,
                 nature: file.nature,
                 addedOn: file.addedOn?.toISOString() || undefined,
+                submitter: locDescription.requesterAddress // TODO Change me
```

Comment:
Currently, the submitter is the owner, not the requester.

Reply:
Good point. Fixed.